code/game/mob/living/simple_animal/friendly/static.dm

```diff
@@ -181,18 +181,25 @@
 			done = TRUE
 	spawn(600) // 1 minute 
 		check_food()
-/obj/structure/anthill/attackby(var/obj/item/stack/W as obj, var/mob/living/human/H as mob)
+
+/obj/structure/anthill/attackby(var/obj/item/weapon/W as obj, mob/user as mob)
 	if (istype(W, /obj/item/weapon/branch))
 		var/obj/item/weapon/branch/B = W
-		H.visible_message("[H] starts poking inside the anthill with the stick.")
-		if (do_after(H, 120, src))
-			if (prob(40))
-				H << "You get some ants on your stick."
+		user.visible_message(SPAN_WARNING("You start poking inside \the [src] with the stick."), SPAN_WARNING("[user] starts poking inside \the [src] with the stick."))
```

Comment:
If this is /mob/visible_message then the first parameter is what people see in third-person and the second is what the mob doing the action will see.

Reply:
RRRiiiiight....